Automattic/jetpack
# Boost: Fix Image CDN quality controls UI showing for the free version

## Proposed changes:


* Fix showing quality control settings instead of upgrade CTA for the free version.

	<details><summary>Before</summary>
	
	![CleanShot 2024-02-08 at 14 28 20](https://github.com/Automattic/jetpack/assets/11799079/ccf85eab-1fdc-4807-a7d1-0aeaf9762d87)
	
	</details>
	
	<details><summary>After</summary>
	
	![CleanShot 2024-02-08 at 14 28 53](https://github.com/Automattic/jetpack/assets/11799079/8d34142f-a954-4631-b96a-7a5f114862eb)
	
	</details>



### Other information:


## Jetpack product discussion



n/a

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Setup Boost (free);
* Make sure the upgrade CTA is showing instead of the quality controls UI.
